docs/stories/ssl/making-tls-work-locally.md

```diff
@@ -15,14 +15,14 @@ Next, we'll generate a certificate and put it in the right directory for Mage2Do
 
 ```bash
 mkcert -key-file magento2.test.key -cert-file magento2.test.pem magento2.test
-mv magento2.test.key nginx/certs/magento2.test.key && mv magento2.test.pem nginx/certs/magento2.test.pem
+mkdir -p nginx/certs && mv magento2.test.key nginx/certs/magento2.test.key && mv magento2.test.pem nginx/certs/magento2.test.pem
 ```
 
 ## Enabling Local Cert Mounting with Mage2Docker
 Now, simply append the `docker-compose.tls.yml` path to your `.env` file and bring your containers up `docker-compose up`.
 
 ```
-COMPOSE_FILE=docker-compose.yml:...:docker-compose.tls.yml
+COMPOSE_FILE=docker-compose.yml:...:path/to/docker-compose.tls.yml
```

Comment:
This path is pre-determined and should simply be the `compose` path.

Reply:
changed